validation/manifest.go

```diff
@@ -10,21 +14,116 @@ import (
 
 // ValidateManifest does validation for manifest
 func ValidateManifest(m manifest.Manifest) error {
+	if m == nil {
```

Comment:
`len(m) == 0`?

Reply:
That should work, but its a little strange since it isn't immediately obvious that `Manifest` is a slice.

Reply:
I agree.  okay.  but are we checking for `len(m) == 0` another way?